diff --git a/samples/playr-tutorial/app/controllers/Application.scala b/samples/playr-tutorial/app/controllers/Application.scala
index 3e759bb..4ddeacf 100644
--- a/samples/playr-tutorial/app/controllers/Application.scala
+++ b/samples/playr-tutorial/app/controllers/Application.scala
@@ -5,6 +5,7 @@ import play.api.mvc._
 import twentysix.playr.RestApiRouter
 import twentysix.playr.RestResourceRouter
 import twentysix.playr.GET
+import twentysix.playr.swagger.SwaggerRestDocumentation
 
 object Application extends Controller {
 
@@ -18,4 +19,6 @@ object Application extends Controller {
   val api = RestApiRouter()
     .add(new RestResourceRouter(ColorController))
     .add("crm" -> crmApi)
+
+  val apidocs = new SwaggerRestDocumentation(api)
 }
\ No newline at end of file
diff --git a/samples/playr-tutorial/build.sbt b/samples/playr-tutorial/build.sbt
index 31120ec..1b7ba18 100644
--- a/samples/playr-tutorial/build.sbt
+++ b/samples/playr-tutorial/build.sbt
@@ -8,7 +8,6 @@ scalacOptions += "-language:reflectiveCalls"
 
 libraryDependencies += cache
 
-lazy val playr = RootProject(file("../.."))
-
-lazy val playrTutorial = project in file(".") dependsOn playr
+libraryDependencies += "26lights"  %% "playr"  % "0.1.0-SNAPSHOT"
 
+libraryDependencies += "26lights"  %% "playr-swagger"  % "0.1.0-SNAPSHOT"
diff --git a/samples/playr-tutorial/conf/routes b/samples/playr-tutorial/conf/routes
index 47b55cd..ef35ff9 100644
--- a/samples/playr-tutorial/conf/routes
+++ b/samples/playr-tutorial/conf/routes
@@ -3,4 +3,5 @@
 # ~~~~
 
 ->      /api                        controllers.Application.api
+->      /api-docs                   controllers.Application.apidocs
 
